### ISSUE TITLE
vc.video() with MediaStream from getUserMedia? (web cam)

### ISSUE THREAD

**Issue by zoanyway** [bbc/VideoContext#148]
Has anyone had any success using the webcam as a source for VideoContext.video(...)? I can render the webcam into a <video> HTML element, and even tried using that video element as the argument to vc.video(...) but I never get any output to the destination canvas.

**Comment by johnny19941216** [comment]
you can add a video element
```js

// 想要获取一个最接近 1280x720 的相机分辨率
var constraints = { audio: true, video: { width: 1280, height: 720 } }; 

navigator.mediaDevices.getUserMedia(constraints)
.then(function(mediaStream) {
  var videoEle = document.querySelector('video');
  videoEle.srcObject = mediaStream;

  vc.video(videoEle)
})
.catch(function(err) { console.log(err.name + ": " + err.message); }); // 总是在最后检查错误

```

**Comment by PTaylour** [comment]
Do you have a codesandbox of where you're up to so far? I'll can give it a go

**Comment by zoanyway** [comment]
Thanks for the input guys. For this project/deadline (an interactive and sensor-driven big screen video presentation) I just had to punt, and went with OBS, which meant I had to fumble my way through Python instead of the JS/ES in which I'm already fluent.  I think VideoContext is an awesome tech, and I'm quite sure I'll use it in the future - especially if I can figure out the live/local webcam source.
